src/main/java/modify_event_use_case/ModifyEventDataAccess.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Data Access Class. Interacts with the in-memory week to "save" modifications.
+ * Author: Daniel Livshits
+ * Created: Nov 19, 2022
+ * Modified by:
+ * Last Modified: Nov 19, 2022
+ */
+package modify_event_use_case;
+
+import data_access.WeekDataAccess;
+import data_access.DataAccessEvent;
+
+import java.time.LocalTime;
+import java.util.HashMap;
+import java.util.Objects;
+
+public class ModifyEventDataAccess extends WeekDataAccess implements ModifyEventDsGateway{
+    @Override
+    public void save(ModifyEventDsInputData dataAccessInput){
```

Comment:
Kindly use dsInputData instead of dataAccessInput, as compliant with our convention,

Reply:
Thank you for pointing this out. I will fix it as soon as possible.